package.json

```diff
@@ -111,40 +109,35 @@
     ]
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^5.11.9",
-    "@testing-library/react": "^11.2.3",
-    "@testing-library/user-event": "^12.6.2",
+    "@testing-library/dom": "^7.31.0",
+    "@testing-library/jest-dom": "^5.12.0",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^13.1.9",
     "@types/classnames": "^2.2.11",
     "@types/i18next-fs-backend": "^1.0.0",
-    "@types/jest": "^26.0.20",
-    "@types/node": "^14.14.22",
+    "@types/jest": "^26.0.23",
+    "@types/node": "^15.6.1",
```

Comment:
I'm wondering if, since we are using node 14.7, would be better to use the types for that version, since we can have wrong information and types from methods that exists on node 15 and dont exists on node 14.